boards/nordic/nrf9280pdk/nrf9280pdk_nrf9280-ipc_conf.dtsi

```diff
@@ -1,23 +1,23 @@
 /*
- * Copyright (c) 2024 Nordic Semiconductor ASA
+ * Copyright (c) 2025 Nordic Semiconductor ASA
```

Comment:
nit: are your changes really major enough to bump the year? True for all the copyright year bump, tbh looks like noise in the pr